### PR TITLE
K8SPG-79 Merging upstream master branch

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/796e09ec64cba6622476a448f76e508bdad43753
image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-796e09ec-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/36ffb80c1f1740df393bf0ba1d9e795fee9d840d
image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-36ffb80c-pgo-deployer`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (hors) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: please remove this issue templat

**Review by cap1984 — COMMENTED** [review]
I guess we should add some commits missing 

```
4333c74b161ccca3e40582776a05753b0c4d9198 (HEAD, tag: v4.7.0) Bump Prometheus to v2.26.1
4e04a6dcef107ae730fd5de49274d7aafad266f2 Ensure label correctness when creating clusters with API
38512a6ee3b6952656cfd55fa1d869197c581770 Version bump v4.7.0
2504746dfc67a3ca700a73914845e2b9b3fae9c0 Fixes to some URLs
cdcf3c7699e4131cc09d05b50031fec69a3c09de Consider posix,s3 & posix,gcs During Upgrade
29fa81ceb157acd3be775235b3afcf7f37594df4 (tag: v4.7.0-rc.1) Fix panic when parsing backup options with extra space
cb89f0b5ab8b69ae974da5d460b4b3f75fc4ee6f Bump v4.7.0-rc.1
8a48507632ff57dae6f17c2a453e298ed2d4d597 Several documentation updates
081b9d366abe0f97aa5c1828953765aca9e6193e Add explicit disable_fsgroup to OpenShift 3.11 manifest
671856deb5795f28651603ca9621a07466b7d26f Fix typos in casing of backrest parameters
9029cf8693a3983f83203fdf1fb72adda9e049b3 Ensure custom labels appear in "pgo show cluster" command
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/83e5024de67fc932867cad47d41b70bad41f2331
image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-83e5024d-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/2ce880341b3762bbc9e117629a550dfd5ba2f4eb
image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-2ce88034-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/3eccd8a851e2fc1fe89898624c95ba4ac2aa4248
image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-3eccd8a8-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/07043211be8c5c80c7cd12256bb3fef2425e5ac3
image: `perconalab/percona-postgresql-operator:PR-64-07043211-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-07043211-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-07043211-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-07043211-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-07043211-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-07043211-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/3a3445e4a11f89a4ee0185b5b10766b63ac5f560
image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/64/commits/3a3445e4a11f89a4ee0185b5b10766b63ac5f560
image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-64-3a3445e4-pgo-deployer`

**Review by hors — APPROVED** [review]

**Pull request merged**